README.md

```diff
@@ -78,23 +78,23 @@ The LoopBack community has created and supports a number of additional connector
 
 StrongLoop provides a number of example applications that illustrate various key LoopBack features. In some cases, they have accompanying step-by-step instructions (tutorials).
 
-See [loopback-example](https://github.com/strongloop/loopback-example) for details.
```

Comment:
Please change the link title too - remove `loopback-example`, use something like `examples at loopback.io` instead.